### PR TITLE
feat: Change new toolbar mode switching buttons labels for ADP 

### PR DESCRIPTION
**Pull request by Dannysht**
Add UI Adaptation and Navigation button labels in toolbar in ADP case

issue: #2067

### CODE REVIEW

**Review by GDamyanov — CHANGES_REQUESTED** [review]
Please add reference to the issue in the PR description.

**Review by voicis — CHANGES_REQUESTED** [review]
Please add unit tests for this scenario.

**Comment by Dannysht** [comment]
> Member

File: packages/control-property-editor/src/slice.ts [hunk 1]
```diff
@@ -133,7 +133,30 @@ export const initialState: SliceState = {
...
     appMode: 'adaptation',
-
+    changeStack: {
+        canUndo: false,
+        canRedo: false
+    },
+    canSave: false,
+    isAppLoading: true
+};
+export const adpState: SliceState = {
```
Comment (voicis) [inline packages/control-property-editor/src/slice.ts:143]: This code is only used for tests and it should not be here. Please remove it.

File: packages/control-property-editor/test/unit/toolbar/ModeSwitcher.test.tsx [hunk 2]
```diff
@@ -38,3 +38,28 @@ test('renders ModeSwitcher', () => {
     editBtn.click();
     expect(dispatch).toBeCalledWith(setAppMode('adaptation'));
 });
+
+test('renders ModeSwitcher with changed button names for ADP', () => {
+    const { dispatch, store } = render(<ModeSwitcher />, { initialState: adpState });
```
Comment (voicis) [inline packages/control-property-editor/test/unit/toolbar/ModeSwitcher.test.tsx:43]: You don't need to create a copy of an initial state, you can just set the scenario like this `store.dispatch(setProjectScenario('ADAPTATION_PROJECT'));`.

**Review by voicis — CHANGES_REQUESTED** [review]

File: packages/control-property-editor/test/unit/toolbar/ModeSwitcher.test.tsx (hunk 2 above)
Comment (Dannysht) [reply packages/control-property-editor/test/unit/toolbar/ModeSwitcher.test.tsx]: Done in [6c93187](https://github.com/SAP/open-ux-tools/pull/2068/commits/6c9318770081a6025469af63e0a5939a2a2ef178)

File: packages/control-property-editor/src/slice.ts (hunk 1 above)
Comment (Dannysht) [reply packages/control-property-editor/src/slice.ts]: Done in [6c93187](https://github.com/SAP/open-ux-tools/pull/2068/commits/6c9318770081a6025469af63e0a5939a2a2ef178)

**Review by voicis — DISMISSED** [review]
Thanks @Dannysht!
- Simple code change
- Review comments addressed
- Good code coverage

There is no changeset for preview middleware, so it will only be available with the next release of preview middleware.

**Review by voicis — DISMISSED** [review]

**Review by nikmace — DISMISSED** [review]
Code looks good
Tests present
Changeset present
Did not test manually

**Review by GDamyanov — APPROVED** [review]
Small and simple change, code looks good.
Great test coverage,
Did not tested manually.

**Review by nikmace — APPROVED** [review]
Re-approving

**Review by voicis — APPROVED** [review]

**Pull request merged**